AutonomousDemo/Loop.cs

```diff
@@ -10,36 +10,26 @@ namespace AutonomousDemo
     public class Loop
```

Comment:
Probably should just keep building out this class until you can run it and [something happens].